packages/components/src/utils/font.js

```diff
@@ -14,5 +9,5 @@ import FONT from './font-values';
  * @return {string} Font rule value
  */
 export function font( value ) {
-	return get( FONT, value, '' );
+	return FONT[ value ] ?? '';
```

Comment:
Unfortunately, this won't work; `value` could contain dots that define a nested path, for example:

https://github.com/WordPress/gutenberg/blob/19efda38e0f4c6784863162f3482b685a9519059/packages/components/src/base-control/styles/base-control-styles.ts#L65

We'll need to parse and account for those when retrieving the field value. You could borrow the `getValueFromObjectPath()` helper from #48491 which does essentially that.

Reply:
I don't think it's a path. The keys of the `FONT` map contain dots, too. The JSDoc mentions its expecting one of those keys. Think we're good in this case.

Reply:
Glad this was a false alarm. Thanks for double-checking. I've confirmed that you're correct. 👍